### PR TITLE
Add new keep annotations for testing

### PR DESCRIPTION
**Pull request by dsvoronin**
Run `SyntheticConstructorCaseTest.syntheticConstructorCall` test.

```
java.lang.NoSuchMethodError: No direct method <init>(ILjava/lang/String;Ljava/lang/Throwable;ILkotlin/jvm/internal/DefaultConstructorMarker;)V in class Lcom/avito/android/ui/SyntheticConstructorCase$Two; or its super classes (declaration of 'com.avito.android.ui.SyntheticConstructorCase$Two' appears in /data/app/com.avito.android.ui-q5NbB9yNay4ZZmHPvAnKKw==/base.apk)
        at com.avito.android.ui.test.SyntheticConstructorCaseTest.syntheticConstructorCall(SyntheticConstructorCaseTest.kt:17)
```

https://r8-docs.preemptive.com/
https://issuetracker.google.com/issues/143419092

### CODE REVIEW

**Comment by dsvoronin** [comment]
https://r8-docs.preemptive.com/
https://issuetracker.google.com/issues/143419092

run `SyntheticConstructorCaseTest.syntheticConstructorCall`

**Comment by dsvoronin** [comment]
slackhq/keeper/pull/21#issuecomment-583070627

File: docs/content/test/TestMinimized.md [hunk 1]
```diff
@@ -50,9 +50,11 @@ If this is your case, add these resources to `res/raw/keep.xml` like this:
            tools:keep="@layout/page_object*"/>
 ```
 
-### Try latest stable version of R8
+### X already has a mapping
 
-Some issues probably solved in new version of r8. For example: ["already has a mapping" one](https://issuetracker.google.com/issues/122924648)
+[Issue](https://issuetracker.google.com/issues/122924648)
```
Comment (eugene-krivobokov) [inline docs/content/test/TestMinimized.md]: I'd prefer to see an issue number in the link:
`[Issue #122924648](https://issuetracker.google.com/issues/122924648)`

**Review by sergioboi — APPROVED** [review]

File: docs/content/test/TestMinimized.md [hunk 2]
```diff
@@ -50,9 +50,11 @@ If this is your case, add these resources to `res/raw/keep.xml` like this:
...
+
+Solved in a new version of r8.
```
Comment (eugene-krivobokov) [inline docs/content/test/TestMinimized.md]: which one? Without exact version, it's hard to check

**Review by eugene-krivobokov — APPROVED** [review]

File: docs/content/test/TestMinimized.md (hunk 2 above)
Comment (dsvoronin) [reply docs/content/test/TestMinimized.md]: i don't know, it's kinda hard to find

Comment (dsvoronin) [reply docs/content/test/TestMinimized.md]: it is probably 1.6.x, because reproduced in 1.5.x

**Pull request merged**